config/software/ncurses.rb

```diff
@@ -69,9 +69,8 @@
     patch source: "patch-aix-configure", plevel: 0, env: patch_env
   end
 
-  if mac_os_x? ||
-    # Clang became the default compiler in FreeBSD 10+
-    (freebsd? && ohai['os_version'].to_i >= 1000024)
+  # Clang became the default compiler in FreeBSD 10+
+  if mac_os_x? || (freebsd? && ohai['os_version'].to_i >= 1000024)
```

Comment:
I know this is a mechanical fix of style issues.....but since we are touching this, perhaps it would help to provide a:

```
using_clang?
```

function that would hold these rules?